src/formatDate.js

```diff
@@ -50,7 +50,48 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const symbolA = fromFormat[fromFormat.length - 1];
+  const symbolB = toFormat[toFormat.length - 1];
+
+  const newDateArray = date.split(symbolA).join(symbolB).split(symbolB);
+  const newToFormat = toFormat.filter(dateExpression =>
+    dateExpression !== dateExpression[dateExpression.length - 1]);
+  const newFromFormat = fromFormat.filter(dateExpression =>
+    dateExpression !== dateExpression[dateExpression.length - 1]);
+
+  const obj = {};
```

Comment:
Use more descriptive names for the objects, e.g. 'dateObj' and 'formattedDateObj'